Apps/phtaniumrest/taniumrest_connector.py

```diff
@@ -664,6 +695,7 @@ def initialize(self):
         self._username = config['username']
         self._password = config['password']
         self._verify = config['verify_server_cert']
+        self._percentage = config['results_percentage']
```

Comment:
It looks like you initialize this value, but never use it.

Reply:
You're right. I access this value on line 434 using config.get
Removing now.